package Exception_Handling;

public class Example3
{
	public static void main(String[] args) {
		
	
	int a=10;
	int b=0;
	int div;
	
	try
	{
		div=a/b;
	}
	catch(ArithmeticException e)
	{
		System.out.println("exception handled");
	}
	
	System.out.println("hello");
	
	System.out.println("i am bhushan");
		
	}
	
	
	}

